FEATURE_TEMPLATE.md

```diff
@@ -0,0 +1,21 @@
+### User Story
```

Comment:
btw I just read https://help.github.com/articles/creating-issue-templates-for-your-repository/
I think we should configure it like this and not add these files, wdyt?

Reply:
Yes, but if you configure it that way, you will be asked which template to use when you create an issue. Feel free to try it. I haven't seen it in action yet, just found the GitHub documentation.

Reply:
You need to have the correct file names - this is why I opened it - to force user go through a checklist :). I am sure about ISSUE and PR but not sure about Feature. We need to test that .